File: 21-10-24_Split_the_Array.java
class Solution 
{

    public static int countgroup(int arr[])
    {
        
        int totalXor = arr[0];
        int n = arr.length;
        
        for (int i = 1; i < n; i++) 
        {
            totalXor = totalXor ^ arr[i];
        }
        
        if (totalXor != 0) 
        {
            return 0;
        }
        
        return (1 << (n - 1)) - 1;
        
    }
}

File: 21-10-24_Split_the_Array.py
class Solution:
	def countgroup(self,arr): 
	   
        total_xor = arr[0]
        n = len(arr)
    
        for i in range(1, n):
            total_xor ^= arr[i]
    
        if total_xor != 0:
            return 0
    
        return (1 << (n - 1)) - 1

		

